config/auth.php

```diff
@@ -102,7 +102,7 @@
     'passwords' => [
         'users' => [
             'provider' => 'users',
-            'table' => 'password_resets',
+            'table' => 'password_reset_tokens',
```

Comment:
This is interesting... I don't see this table in the DB and we don't handle any passwords - is this necessary or can we just remove this section?

Reply:
I updated it to be consistent with the 10.x repo since it is part of our code base, but you're right... we could probably just get rid of it. Thoughts?

Reply:
I would say let's see if it can be removed. I don't see a case where we'll ever need to store passwords, so no use in keeping it

Reply:
Agreed - removed.